pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -154,7 +154,9 @@ def results(self) -> Optional[Union[_EVALUATE_OUTPUT, _PREDICT_OUTPUT]]:
         """
         return self._results
 
-    def load_checkpoint_file(self, checkpoint_path: Union[str, Path]) -> Dict[str, Any]:
+    def load_checkpoint(self, checkpoint_path: Union[str, Path]) -> Dict[str, Any]:
+        if self.on_gpu:
+            torch.cuda.empty_cache()
```

Comment:
```suggestion
        torch.cuda.empty_cache()
```
not needed. call to `empty_cache` is a no-op if cuda is not available